### PR TITLE
Update build stuff in deploy.md

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

**Comment by abdonrd** [comment]
Thanks @katejeffreys! 👏

Same comment as https://github.com/PolymerElements/polymer-starter-kit/pull/1044#issuecomment-313220222.

**Comment by arthurevans** [comment]
Yeah, I think we should be using `es5-bundled` for both (Firebase docs are currently using `es6-bundled` which won't work for older browsers.)

File: app/2.0/start/toolbox/deploy.md [hunk 1]
```diff
@@ -9,21 +9,43 @@ In this step, you'll deploy your application to the web.
...
+    "preset": "es6-bundled"
+  },
+  {
+    "preset": "es6-unbundled"
+  }
+]
+...
+```
+
+The builds will be output to a subdirectory under the `build/` directory as follows:
```
Comment (arthurevans) [inline app/2.0/start/toolbox/deploy.md]: a subdirectory => subdirectories

Comment (arthurevans) [inline app/2.0/start/toolbox/deploy.md]: Or folder/subfolders...  I think we mostly use "folder" in the docs (and you do below, so probably stay consistent.)

File: app/2.0/start/toolbox/deploy.md [hunk 2]
```diff
@@ -58,36 +80,38 @@ and create a new project
 
 1.  `cd` into the main folder for your app (e.g. `my-app/`).
 
-1. Create an `app.yaml` file with the following contents:
+1. Create an `app.yaml` file with the following contents (replace <code><var>build-folder-name</var></code> with the name of your build folder):
```
Comment (arthurevans) [inline app/2.0/start/toolbox/deploy.md]: I think both of this example and the firebase one should use `es5-bundled` throughout. Neither of the other options will run on all browsers. 

You really want a custom, dynamic serving solution (like `prpl-server-node`) if you're going to serve either of the other builds.

File: app/2.0/start/toolbox/deploy.md [hunk 3]
```diff
@@ -60,34 +82,36 @@ and create a new project
 
 1. Create an `app.yaml` file with the following contents:
 
-    ```
+<pre>
+  <code>
```
Comment (abdonrd) [inline app/2.0/start/toolbox/deploy.md]: Why `pre` `code` instead of ` ``` `?

File: app/2.0/start/toolbox/deploy.md [hunk 4]
```diff
@@ -59,36 +81,35 @@ and create a new project
 1.  `cd` into the main folder for your app (e.g. `my-app/`).
 
 1. Create an `app.yaml` file with the following contents:
-
-    ```
+    ````
```
Comment (abdonrd) [inline app/2.0/start/toolbox/deploy.md]: Ups! Only three ` :)

File: app/2.0/start/toolbox/deploy.md [hunk 5]
```diff
@@ -59,36 +81,35 @@ and create a new project
...
 
     - url: /.*
-      static_files: build/default/index.html
-      upload: build/default/index.html
+      static_files: build/es5-bundled/index.html
+      upload: build/es5-bundled/index.html
       secure: always
-    ```
-
+    ````
```
Comment (abdonrd) [inline app/2.0/start/toolbox/deploy.md]: Ups! Only three ` :)

File: app/2.0/start/toolbox/deploy.md [hunk 6]
```diff
@@ -59,36 +81,35 @@ and create a new project
...
+
```
Comment (abdonrd) [inline app/2.0/start/toolbox/deploy.md]: White spaces.

File: app/2.0/start/toolbox/deploy.md (hunk 4 above)
Comment (abdonrd) [inline app/2.0/start/toolbox/deploy.md]: Also, we should add yaml?

**Comment by katejeffreys** [comment]
@arthurevans @abdonrd Good to go?

File: app/2.0/start/toolbox/deploy.md [hunk 7]
```diff
@@ -170,7 +181,7 @@ guide](https://www.firebase.com/docs/hosting/quickstart.html).
         "rules": "database.rules.json"
       },
       "hosting": {
-        "public": "build/default",
+        "public": "build/es5-bundled",
```
Comment (abdonrd) [inline app/2.0/start/toolbox/deploy.md]: Minor: Update to `"build/es5-bundled/"`.
In the PSK (and here in the "Build for deployment" we always use `/` to 'close' a folder name.

**Comment by abdonrd** [comment]
Thanks @katejeffreys & @arthurevans! 🎉

**Pull request merged**